go.mod

```diff
@@ -41,6 +41,7 @@ require (
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/moby/sys/signal v0.7.0
 	github.com/moby/sys/symlink v0.2.0
+	github.com/montanaflynn/stats v0.6.6
```

Comment:
Can we make integration/benchmark/ its own module, so that this dependency doesn't become a dependency of containerd itself? Similar to what we do for `integration/client` https://github.com/containerd/containerd/blob/main/integration/client/go.mod

Reply:
oh, I see a similar comment was left above https://github.com/containerd/containerd/pull/7379#discussion_r966131197 sorry for duplicating that